spec/services/complete_moab_handler_update_version_spec.rb

```diff
@@ -34,9 +30,9 @@
 
     context 'in Catalog' do
       before do
-        create(:preserved_object, druid: druid, current_version: 2, preservation_policy: default_prez_policy)
-        po.complete_moabs.create!(
-          version: po.current_version,
+        v2 = create(:preserved_object, druid: druid, current_version: 2, preservation_policy: default_prez_policy)
+        v2.complete_moabs.create!(
+          version: v2.current_version,
```

Comment:
create a different preserved object instance for this test to avoid clashing with other test objects